Doc/library/functions.rst

```diff
@@ -1570,7 +1570,7 @@ are always available.  They are listed here in alphabetical order.
    :exc:`RuntimeError`.
 
 
-.. function:: reversed(seq)
+.. function:: reversed(sequence, /)
```

Comment:
I'd leave the param name untouched and instead consider updating the docstring.

```suggestion
.. function:: reversed(seq, /)
```

Reply:
In this case at least ISTM `sequence` reads clearer than `seq`; beginners and particularly non-native speakers may not immediately assume `seq` is short for `sequence`.